modules/build/src/test/scala/scala/build/tests/TestLogger.scala

```diff
@@ -24,7 +24,7 @@ case class TestLogger(info: Boolean = true, debug: Boolean = false) extends Logg
     if (debug)
       System.err.println(s)
 
-  def log(ex: BuildException): Unit =
+  def log(ex: BuildException, severity: Severity): Unit =
```

Comment:
No need to pass a severity here, we should handle warnings later, in a more specific way.